huaweicloud/services/apig/resource_huaweicloud_apig_group.go

```diff
@@ -355,8 +345,8 @@ func resourceGroupRead(_ context.Context, d *schema.ResourceData, meta interface
 		d.Set("description", resp.Description),
 		d.Set("url_domains", flattenUrlDomain(resp.UrlDomians)),
 		d.Set("domain_access_enabled", resp.SlDomainAccessEnabled),
-		d.Set("registration_time", resp.RegistraionTime),
```

Comment:
Please keep this field setting operation. The current processing is an incompatible change.

Reply:
sure

Reply:
done.